### PR TITLE
FEAT: add export gds comps xml function

### PR DESCRIPTION
**Pull request by felipeescastro**

### CODE REVIEW

File: src/pyedb/dotnet/edb.py [hunk 1]
```diff
@@ -4598,3 +4598,51 @@ def definitions(self):
...
+
+        if not comps_to_export:
+            comps_to_export = self.components.components
+
+        for comp in comps_to_export:
+            ocomp = self.components.components[comp]
+            fo_txt += "\t<GDS_COMPONENT>\n"
+            for pin in ocomp.pins:
+                pins_position_unit = unit_converter(ocomp.pins[pin].position, output_units=gds_comps_unit)
+                fo_txt += '\t\t<GDS_PIN Name="{}" x="{}" y="{}" Layer="{}"/>\n'.format(
```
Comment (svandenb-dev) [inline src/pyedb/dotnet/edb.py:4631]: I would use xml.etree instead of working on raw string

**Review by svandenb-dev — APPROVED** [review]
LGTM

**Review by SMoraisAnsys — APPROVED** [review]

**Comment by felipeescastro** [comment]
> @felipeescastro Could you see if the test isn't taking too long to run ? It seems that the time required to run the CICD (for tests) has doubled in this PR.

@SMoraisAnsys I'm not sure. The code runs smoothly locally and the unittest I added is supposed to be as fast as any other test in the test file.

**Comment by SMoraisAnsys** [comment]
> > @felipeescastro Could you see if the test isn't taking too long to run ? It seems that the time required to run the CICD (for tests) has doubled in this PR.
> 
> @SMoraisAnsys I'm not sure. The code runs smoothly locally and the unittest I added is supposed to be as fast as any other test in the test file.

Simon fixes the bug, have a look at his changes :)

**Pull request merged**